eclipse-temurin/content.md

```diff
@@ -12,6 +12,15 @@ The Eclipse Temurin project provides code and processes that support the buildin
 
 JRE images are available for all versions of Eclipse Temurin but it is recommended that you produce a custom JRE-like runtime using `jlink` (see usage below).
 
+# Can I add my internal CA certificates to the truststore?
+
+Yes! Add your certificates to `/certificates` inside the container (e.g. by using a volume) and set the environment variable `USE_SYSTEM_CA_CERTS` on the container to any value. With Docker CLI this might look like this: 
+```
+docker run -v $(pwd)/certs:/certificates/ -e USE_SYSTEM_CA_CERTS=1 %%IMAGE%%:11 
+```
```

Comment:
Needs a newline between elements and a language to use a fenced code block. I think this should be enough to make the `markdownfmt` test green:
````suggestion

```console
$ docker run -v $(pwd)/certs:/certificates/ -e USE_SYSTEM_CA_CERTS=1 %%IMAGE%%:11 
```
````